app/konnect/org-management/saml-idp.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Set up SSO with SAML 2.0
+content_type: how-to
+---
+
+As an alternative to {{site.konnect_saas}}’s native authentication, you can set
+up single sign-on (SSO) access to {{site.konnect_short_name}} through
+an identity provider (IdP) with
+SAML 2.0.
+This authentication method allows your users to log in to {{site.konnect_saas}}
+using their IdP credentials, without needing a separate login.
+
+## Prerequisites
+
+* {{site.konnect_short_name}} must be added to your IdP as an application
+
+## Set up SSO in {{site.konnect_short_name}}
+
+1. In [{{site.konnect_saas}}](https://cloud.konghq.com/login), click {% konnect_icon organizations %} **Organization**, and then **Auth Settings**.
+
+2. Click **Configure provider** for **SAML**. 
+
+3. Paste the **Metadata URL** from your IdP in the **Identity Provider Metadata URL** box or if you have the **Metadata XML** paste it in the **Identity Provider Metadata XML** box
+
+4. In the **Organization Login Path** box, enter a unique string. For example: `examplepath`.
+
+   {{site.konnect_short_name}} uses this string to generate a custom login
+   URL for your organization.
+
+   Requirements:
+    * The path must be unique *across all {{site.konnect_short_name}} organizations*.
+      If your desired path is already taken, you must to choose another one.
+    * The path can be any alphanumeric string.
+    * The path does not require a slash (`/`).
+
+5. Click **Save**.
```

Comment:
Are there any additional steps required on the IdP's side of the world to run through before testing the configuration? I'm thinking about configuring the SP medatada or entity ID?